src/manifest.ts

```diff
@@ -1,8 +1,8 @@
+import { promises as fs } from 'fs'
```

Comment:
Maybe it'll be better and more consistent with your other projects to use `import { promises as fsPromises } from 'fs'`?